Connectors 25.06 Release note CTOR-1641 (#4415)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md

## 2025

### Juin

<Tabs groupId="sync">
<TabItem value="New connectors" label="New connectors">

- [**Juniper M-Series Netconf**](../procedures/network-routers-juniper-mseries-netconf.md) - Initial release of Juniper M-series with Netconf.

</TabItem>
<TabItem value="Enhancements" label="Enhancements">

> - [**F5 BigIP**](../procedures/network-loadbalancers-f5-bigip-snmp.md) - Added new modes **certificates**, from a [The Watch idea](https://thewatch.centreon.com/ideas/monitor-certificate-with-f5-bigip-3074).

Reviewer: ```suggestion
- [**F5 BigIP**](../procedures/network-loadbalancers-f5-bigip-snmp.md) - Added new **certificates** mode, from a [The Watch idea](https://thewatch.centreon.com/ideas/monitor-certificate-with-f5-bigip-3074).
```

- [**Juniper Mag**](../procedures/network-firewalls-juniper-mag-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper SA**](../procedures/network-firewalls-juniper-sa-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper SRX**](../procedures/network-firewalls-juniper-srx-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper SSG**](../procedures/network-firewalls-juniper-ssg-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper Trapeze**](../procedures/network-juniper-trapeze-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Nmap CLI**](../procedures/applications-nmap-cli.md) - Enhanced secure execution of commands.
- [**Office365 OneDrive**](../procedures/cloud-microsoft-office365-onedrive.md) - Added template for **users-activity** and service discovery rule for **site-usage**.

> - [**Pure Storage FlashArray Rest API v2**](../procedures/hardware-storage-purestorage-flasharray-v2-restapi.md) - Added performances counters in **arrays** mode, based on community contribution [PR 5355](https://github.com/centreon/centreon-plugins/pull/5355).

Reviewer: ```suggestion
- [**Pure Storage FlashArray Rest API v2**](../procedures/hardware-storage-purestorage-flasharray-v2-restapi.md) - Added performance counters in **arrays** mode, based on community contribution [PR 5355](https://github.com/centreon/centreon-plugins/pull/5355).
```

- [**Qnap**](../procedures/hardware-storage-qnap-snmp.md) - Added QuTS Operating System handling for **hardware**, **pools** and **volumes** modes, from community contribution [PR 5564](https://github.com/centreon/centreon-plugins/pull/5564).

</TabItem>
<TabItem value="Bug fixes" label="Bug fixes">

> - [**Office365 OneDrive**](../procedures/cloud-microsoft-office365-onedrive.md) - Fixed service discovery mode for **site-usage**.

Reviewer: ```suggestion
- [**Office 365 OneDrive**](../procedures/cloud-microsoft-office365-onedrive.md) - Fixed service discovery mode for **site-usage**.
```

> - [**Oracle Database**](../procedures/applications-databases-oracle.md) - Fixed segmentation fault with glibc latest version.

Reviewer: ```suggestion
- [**Oracle Database**](../procedures/applications-databases-oracle.md) - Fixed segmentation fault that occurred with glibc's latest version.
```

> - [**Rubrik Rest API**](../procedures/applications-rubrik-restapi.md) - Fixed issue with token renew.

Reviewer: ```suggestion
- [**Rubrik Rest API**](../procedures/applications-rubrik-restapi.md) - Fixed an issue affecting token renewal.
```

> - **VMWare Deamon** - Change rights after install to allows apache (central) and centreon-gorgone (poller) to update the file after a config export.

Reviewer: ```suggestion
- **VMWare Daemon** - Change rights after installation to allow apache (central) and centreon-gorgone (poller) to update the **centreon_vmware.json** file after a configuration export.
```

- [**VMware ESX**](../procedures/virtualization-vmware2-esx.md) - Fixed the Vsan-Cluster-Usage `Cannot get response (timeout received)` error. See the notice below to apply the patch.
- [**VMware vCenter v6**](../procedures/virtualization-vmware2-vcenter-6.md) - Fixed the Vsan-Cluster-Usage `Cannot get response (timeout received)` error. See the notice below to apply the patch.
- [**VMware vCenter**](../procedures/virtualization-vmware2-vcenter-generic.md) - Fixed the Vsan-Cluster-Usage `Cannot get response (timeout received)` error. See the notice below to apply the patch.
- [**VMware VM**](../procedures/virtualization-vmware2-vm.md) - Fixed the Vsan-Cluster-Usage `Cannot get response (timeout received)` error. See the notice below to apply the patch.

> > How to patch the VSAN bug on Debian 11 & 12 and Alma/RHEL/Oracle 9?

Reviewer: ```suggestion
#### How to patch the VSAN bug on Debian 11 & 12 and Alma/RHEL/Oracle 9?
```

> - First, locate the file that must be patched.

Reviewer: ```suggestion
1. Locate the file that must be patched.

```

```bash
find /usr/ -name VICommon.pm
```

> It should be `/usr/local/share/perl5/5.32/VMware/VICommon.pm` on EL9, `/usr/local/share/perl/5.32.1/VMware/VICommon.pm` on Debian 11 and `/usr/local/share/perl/5.36.0/VMware/VICommon.pm` on Debian 12.

Reviewer: ```suggestion

It should be `/usr/local/share/perl5/5.32/VMware/VICommon.pm` on EL9, `/usr/local/share/perl/5.32.1/VMware/VICommon.pm` on Debian 11 and `/usr/local/share/perl/5.36.0/VMware/VICommon.pm` on Debian 12.

```

> - Then run this command to apply the patch (replace "FILE_TO_PATCH" with the result of the previous command):

Reviewer: ```suggestion
2. Run the following command to apply the patch (replace "FILE_TO_PATCH" with the results of the previous command):

```

```bash
patch --backup FILE_TO_PATCH <<'EOF'
--- lib/VMware/share/VMware/VICommon.pm	2025-04-24 17:18:24.938290503 +0200
+++ VICommon.pm	2025-04-24 17:18:18.690399614 +0200
@@ -2319,6 +2319,8 @@
    my $user_agent = $self->{user_agent};
    $user_agent->cookie_jar->as_string
       =~ m/(.*)vmware_soap_session=\"\\\"([0-9a-zA-Z-](.*)+)\\\"\"(.*)/;
+   $user_agent->cookie_jar->as_string
+      =~ m/(.*)vmware_soap_session=[\\\"]*([0-9a-zA-Z-]+)/ unless $2;
    return $2;
 }
EOF
```

> The result of this command should be `patching file /usr/your/path/to/VICommon.pm`.

Reviewer: ```suggestion

The results of this command should be `patching file /usr/your/path/to/VICommon.pm`.
```

> - Then restart the daemon:

Reviewer: ```suggestion
3. Then restart the daemon:

```

- [**Juniper M-Series**](../procedures/network-routers-juniper-mseries-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper Mag**](../procedures/network-firewalls-juniper-mag-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper SA**](../procedures/network-firewalls-juniper-sa-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper SRX**](../procedures/network-firewalls-juniper-srx-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper SSG**](../procedures/network-firewalls-juniper-ssg-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Juniper Trapeze**](../procedures/network-juniper-trapeze-snmp.md) - Reworked plugin path to be more explicit with the initial release of Juniper M-series with Netconf.
- [**Nmap CLI**](../procedures/applications-nmap-cli.md) - Enhanced secure execution of commands.

> - [**Office365 OneDrive**](../procedures/cloud-microsoft-office365-onedrive.md) - Added template for **users-activity** and service discovery rule for **site-usage**.

Author: ```suggestion
- [**Office 365 OneDrive**](../procedures/cloud-microsoft-office365-onedrive.md) - Added template for **users-activity** and service discovery rule for **site-usage**.
```
